core/src/main/java/io/kestra/core/models/flows/input/FloatInput.java

```diff
@@ -1,18 +1,48 @@
 package io.kestra.core.models.flows.input;
 
 import io.kestra.core.models.flows.Input;
+import io.kestra.core.models.validations.ManualConstraintViolation;
+import io.swagger.v3.oas.annotations.media.Schema;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.experimental.SuperBuilder;
 
+import java.util.Set;
 import javax.validation.ConstraintViolationException;
 
 @SuperBuilder
 @Getter
 @NoArgsConstructor
 public class FloatInput extends Input<Float> {
+
+    @Schema(title = "Minimal value.")
+    Float min;
+
+    @Schema(title = "Maximal value.")
+    Float max;
+
     @Override
     public void validate(Float input) throws ConstraintViolationException {
-        // no validation yet
+        if (min != null && input.compareTo(min) < 0) {
```

Comment:
Any reason not to compare in a raw manner (input < min) ?

Reply:
Don't remember why I did this like this for the IntInput but now I prefer to keep it consistent.
We can revisit it later.